app/views/admin/claims/_claims_with_matching_details.html.erb

```diff
@@ -1,5 +1,7 @@
 <table id="claims-with-matches" class="govuk-table govuk-!-margin-bottom-9">
-  <caption class="govuk-table__caption govuk-heading-l">Claims with matching details</caption>
+  <caption class="govuk-table__caption govuk-heading-l">
+    <%= t("admin.duplicate_attributes_title", count: @matching_claims.count, policy: @claim.policy.short_name) %>
```

Comment:
This one is simple enough that maybe we should avoid the indirection of the locale file and inline it instead?

```suggestion
    <%= @claim.policy.short_name %> <%= "claim".pluralize(@matching_claims.count) %> with matching details
```